tools/plugin_checker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM python:3.8.2-slim
+
+RUN apt-get update && \
+    apt-get install wget -y && \
+    wget -O - https://deb.nodesource.com/setup_10.x | bash - && \
```

Comment:
curious. Why do we need nodejs runtime ?

Reply:
Oops, leftover from copy-paste. I'll remove it.